src/helpers.php

```diff
@@ -56,12 +56,12 @@ function backpack_users_have_email()
     function backpack_avatar_url($user)
     {
         $firstLetter = $user->getAttribute('name') ? mb_substr($user->name, 0, 1, 'UTF-8') : 'A';
-        $placeholder = 'https://placehold.it/160x160/00a65a/ffffff/&text='.$firstLetter;
+        $placeholder = 'https://place-hold.it/160x160/00a65a/ffffff/&bold&fontsize=18&text='.$firstLetter;
 
         switch (config('backpack.base.avatar_type')) {
             case 'gravatar':
                 if (backpack_users_have_email()) {
-                    return Gravatar::fallback('https://placehold.it/160x160/00a65a/ffffff/&text='.$firstLetter)->get($user->email);
+                    return Gravatar::fallback('https://place-hold.it/160x160/00a65a/ffffff/&bold&fontsize=18&text='.$firstLetter)->get($user->email);
```

Comment:
```suggestion
                    return Gravatar::fallback($placeholder)->get($user->email);
```

Looks like we hardcoded the URL two times. Over here we should already have this as `$placeholder` right?